internal/module/services/version_manager/version_manager.go

```diff
@@ -42,6 +46,7 @@ var (
 	CreateModuleVersionError               = status.Error(codes.Unknown, "Failed to create module version.")
 	AbortModuleVersionError                = status.Error(codes.Unknown, "Failed to abort module version.")
 	PublishModuleVersionError              = status.Error(codes.Unknown, "Failed to publish module version.")
+	PublishReleaseVersionError             = status.Error(codes.Unknown, "Failed to publish release version.")
 )
```

Comment:
add Development Version here
```suggestion
     DevelopmentVersion = versions.MustParseVersion("0.0.0")
)
```